src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities.mdx

```diff
@@ -0,0 +1,275 @@
+
+* Newer user model: from [one.newrelic.com](https://one.newrelic.com), click **[account dropdown](/docs/new-relic-solutions/get-started/glossary/#account-dropdown) &gt; Administration &gt; Access management &gt; Roles**.  
+* Original user model: from [one.newrelic.com](https://one.newrelic.com), click **[account dropdown](/docs/new-relic-solutions/get-started/glossary/#account-dropdown) &gt; Account settings &gt; Users and roles**.  
+
+## How capabilities relate to roles [#capabilities-roles]
+
+Groupings of capabilities are aggregated into roles. We have pre-built roles for common use cases, and you can build your own custom roles. 
+
+Our pre-built roles work differently depending on which of our [user models](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) you're on: 
+
+* Our newer user model: [learn about roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts). 
+* Our original user model: [learn about roles](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model). 
+
+## Some important tips for using capabilities [#important-points]
+
+Some important points for understanding and using capabilities: 
+
+* **Capabilities can change.** The capabilities we expose in the UI are subject to change and reorganization. The capabilities in this doc were last updated June, 2022.     
+* **Some capabilities have overlapping functionality.** That is why selecting some capability checkboxes will automatically check or uncheck other boxes.
+* **Not all capabilities are exposed.** Some roles have hidden capabilities that are not visible in the UI and not available for addition to a custom role. For example, the **Organization manager** role has capabilities related to managing and configuring organization-level settings, but those are not available for addition to a custom role. 
+* **Capabilities don't affect querying.** Most capabilities apply to New Relic UI experiences and not to querying data. For example, if your capabilities restrict you from accessing the APM UI, you can still query APM-related data using the query builder if you have access to that account. If you require more firm data boundaries for some projects or users, one way to do this is by [adding accounts](/docs/accounts/accounts-billing/account-structure/add-accounts).
+
+## Capability definitions [#capability-definitions]
+
+Here's a composite screenshot of the capabilities we expose in the capabilities UI for the purpose of adding to custom roles. 
+
+<img
+  title="New Relic user capabilities screenshot"
+  alt="New Relic user capabilities UI screenshot"
+  src={newRelicUserCapabilitiesUi}
+/>
+
+To learn more about these capabilities, select a category below. 
```

Comment:
Just to clarify: these collapsers are exhaustive for each feature/product? i.e. you're listing all the relevant capabilities as they pertain to all of NR's features? If yes, I'd err on the side of being more obvious than not here, so there's no doubt that all the relevant info is accounted for/represented. :)

Reply:
there's a relatively small set of capabilities we expose in the capabilities UI. it's far from all the stuff you can do in NR. it's just what they decide to give customers control over for the purposes of creating new custom roles. so it's far from all, in other words. for example, they recently added a capability for controlling who can view/manage a user key, so that hadn't been available for a long time and might have gone for a while without being available. If it's not exposed as a capability, it just means it's probably a fundamental capability tied to all users, or perhaps just to full platform users. It's complicated, basically. but I might explain these things a little better, thanks.